.eslintrc.js

```diff
@@ -99,7 +99,12 @@ module.exports = {
             },
         ],
         "import/newline-after-import": "error",
-        "import/no-unassigned-import": "error",
+        "import/no-unassigned-import": [
+            "error",
+            {
+                allow: ["jest-extended"],
+            },
+        ],
```

Comment:
**praise:** I'm glad you were able to get this working to allow only `jest-extended`.